scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/TaskLauncherInitializer.java

```diff
@@ -274,6 +275,20 @@ public int getReplicationIndex() {
     }
 
     /**
+     * @param genericInformation the generic information of this task
+     */
+    public void setGenericInformation(Map<String, String> genericInformation) {
+        this.genericInformation = genericInformation;
+    }
+
+    /**
+     * @return the generic information of this task
+     */
+    public Map<String, String> getGenericInformation() {
```

Comment:
Should we return a copy of this map so we don t risk that the client will change its state by mistake?